lib/manager/components/LabelEditorModal.js

```diff
@@ -60,7 +60,7 @@ export default class LabelEditorModal extends React.Component<
           <LabelEditor
             label={label}
             onDone={this.close}
-            projectId={projectId}
+            projectId={projectId || label.projectId}
```

Comment:
What is the issue with `projectId` here?

Reply:
Good call on adding some comments thanks! This was the cleanest way I could come up with to support both editing and creating a label. Clarification added in 472950e2